src/panels/config/integrations/ha-integration-card.ts

```diff
@@ -176,17 +189,79 @@ export class HaIntegrationCard extends LitElement {
     `;
   }
 
+  private _logsLink() {
+    return html`<a href="/config/logs"
+      >${this.hass.localize(
+        "ui.panel.config.integrations.config_entry.logs"
```

Comment:
Do we have a way deep link into the logs to filter on <homeassistant.components.{integration}> when it's being retried?

Reply:
Not yet, but we should be able to add a filter in the future. I have improved integration detection in #8912 too.